pom.xml

```diff
@@ -689,6 +689,21 @@
                     <version>1.2.1</version>
                 </plugin>
                 <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-compiler-plugin</artifactId>
+                    <configuration>
+                        <compilerId>javac-with-errorprone</compilerId>
```

Comment:
Maven ships with [plexus javac](http://maven.apache.org/plugins/maven-compiler-plugin/non-javac-compilers.html) and this is just the [plexus javac with errorprone wrapper](http://mvnrepository.com/artifact/org.codehaus.plexus/plexus-compiler-javac-errorprone/2.5) which I'm pretty sure just calls the same compiler that's bundled with Maven underneath.